NitroxPatcher/Patches/Dynamic/SubConsoleCommand_OnConsoleCommand_sub_Patch.cs

```diff
@@ -1,19 +1,41 @@
+using System.Collections.Generic;
 using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using NitroxClient.GameLogic;
 using NitroxModel.Helper;
+using UnityEngine;
 
 namespace NitroxPatcher.Patches.Dynamic;
 
-/// <summary>
-/// Called whenever a Cyclops or Seamoth is spawned. Nitrox already has its own command to spawn vehicles.
-/// This patch is only meant to block the method from executing, causing two vehicles to be spawned instead of one
-/// </summary>
 public sealed partial class SubConsoleCommand_OnConsoleCommand_sub_Patch : NitroxPatch, IDynamicPatch
 {
-    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((SubConsoleCommand t) => t.OnConsoleCommand_sub(default));
+    internal static readonly MethodInfo TARGET_METHOD = Reflect.Method((SubConsoleCommand t) => t.OnConsoleCommand_sub(default));
 
-    public static bool Prefix()
+    /*
+     * REPLACE:
+     * LightmappedPrefabs.main.RequestScenePrefab(text, new LightmappedPrefabs.OnPrefabLoaded(this.OnSubPrefabLoaded));
+     * BY:
+     * LightmappedPrefabs.main.RequestScenePrefab(text, new LightmappedPrefabs.OnPrefabLoaded(SubConsoleCommand_OnConsoleCommand_sub_Patch.WrappedCallback));
+     */
+    public static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
     {
-        Log.InGame(Language.main.Get("Nitrox_CommandNotAvailable"));
-        return false;
+        return new CodeMatcher(instructions).MatchEndForward([
+                                                new CodeMatch(OpCodes.Stfld),
+                                                new CodeMatch(OpCodes.Ldsfld),
+                                                new CodeMatch(OpCodes.Ldloc_0),
+                                                new CodeMatch(OpCodes.Ldarg_0)
+                                            ])
+                                            .SetOpcodeAndAdvance(OpCodes.Ldnull)
+                                            .SetOperandAndAdvance(Reflect.Method(() => WrappedCallback(default)))
+                                            .InstructionEnumeration();
+    }
+
+    public static void WrappedCallback(GameObject prefab)
```

Comment:
Method is named `WrappedCallback `in the patch and `Callback ` in SpawnConsole patch